npm-packages/meteor-babel/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@meteorjs/babel",
   "author": "Meteor <dev@meteor.com>",
-  "version": "7.20.0-beta.4",
+  "version": "7.20.0-beta.5",
```

Comment:
Not related to this PR, but just reminding everyone that we must not forget to change this version from beta to actual version once v3 is released.